src/default-xrp-client.ts

```diff
@@ -44,6 +46,9 @@ export class XRPClientErrorMessages {
   public static readonly xAddressRequired =
     'Please use the X-Address format. See: https://xrpaddress.info/.'
   /* eslint-enable @typescript-eslint/indent */
+
+  public static readonly paymentConversionFailure =
+    'Could not convert payment transaction: (transaction). Please file a bug at https://github.com/xpring-eng/xpringkit'
```

Comment:
fix link: xpring-js vs xpringkit. 

Also maybe link directly to issues:
https://github.com/xpring-eng/Xpring-JS/issues